### ISSUE TITLE
ImageNet Classification with Deep Convolutional Neural Networks

### ISSUE THREAD

**Issue by yj-song131** [GDSC-Ewha-4th/Study-paperReview#5]
* The dataset
   - ImageNet is a dataset of over 15 million labeled images belonging to roughly 22,000 categories.
   - ImageNet consists of variable-resolution images, we down-sampled the images to ad fixed resolution if 256 x 256.
   - We did not pre-processing
* The Architecture
  - 8 layers - 5 conv layer, 3 fc layer
  - Activation Function은 ReLU 사용(faster than their equivalents with tanh units)
* Reducing Overfitting
  - Data Augmentation, Dropout 방법이 있는데 Dropout 사용

* Results
  - ILSVRC-2010 대회에서 top-1 and top-5 ( error rates 37.5% and 17.0%)
  - ILSVRC-2012 대회에서 top-5 test error rate of 15.3%

**Comment by youjun99** [comment]
전공 수업인 데이터와인공지능 수업에서는 인공지능의 사례를 살펴보면서 data augmentation에 대해 언급이 있었는데 반갑네요! 수업에서는 아직까지 방법론에 대해선 자세히 배우지 못했는데 어떤 데이터셋을 활용해서 하는지 방법을 알려주셔서 이미지넷에 대해 간단히 알아볼 수 있었습니다! 수고하셨습니다:)
